src/cmd/tools/archiver.go

```diff
@@ -85,6 +89,7 @@ func init() {
 
 	archiverCmd.AddCommand(archiverCompressCmd)
 	archiverCmd.AddCommand(archiverDecompressCmd)
+	archiverCmd.AddCommand(newVersionCmd("arc", archiverVersion))
```

Comment:
Looking at this again it may be good to name the tool as mholt/archiver so that people can look it up.

Reply:
Done